### PR TITLE
chore: added meta titles and descriptions to pages

### PR DESCRIPTION
**Pull request by atepem**
## Proposed Changes

Added meta titles and descriptions to /opportunities and each /path/[id] page.

### CODE REVIEW

File: components/Meta.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+const SITE_DEFAULT_IMAGE_URL = `${SITE_URL}/images/metatags-image.png`;
+
+const Meta = ({ title, description }) => {
+  const pageTitle = title ? `${title} | ${SITE_TITLE}` : SITE_TITLE;
+
+  return (
+    <Head>
+      <title>{pageTitle}</title>
+
+      <meta name="title" content={pageTitle || SITE_TITLE} />
```
Comment (truemiller) [inline components/Meta.jsx:16]: don't think this tag exists 🤔 
i don't imagine there's any harm if it doesn't though

File: components/Meta.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      <meta name="description" content={description || SITE_DESCRIPTION} />
+
+      <meta property="og:type" content="website" />
+      <meta property="og:url" content={SITE_URL} />
```
Comment (truemiller) [inline components/Meta.jsx:20]: this needs to be dynamic based on the page

File: components/Meta.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      <meta property="og:title" content={pageTitle} />
+      <meta
+        property="og:description"
+        content={description || SITE_DESCRIPTION}
+      />
+      <meta property="og:image" content={SITE_DEFAULT_IMAGE_URL} />
+
+      <meta property="twitter:card" content="summary_large_image" />
+      <meta property="twitter:url" content={SITE_URL} />
```
Comment (truemiller) [inline components/Meta.jsx:29]: same for this site url, needs to be the canonical link of the page it is on

File: components/Meta.jsx (hunk 1 above)
Comment (atepem) [reply components/Meta.jsx]: Sorry, did you mean the meta tag?

Comment (truemiller) [reply components/Meta.jsx]: yeah, correct, `<meta name="title" content={pageTitle || SITE_TITLE} />` 
meta tag with the name "title" isn't used, or parsed by search engines afaik
basically useless code, but harmless nonetheless

File: pages/paths/[id].jsx [hunk 4]
```diff
@@ -72,49 +73,55 @@ const PathDetailPage = () => {
...
-        <Col xs={24} lg={12}>
-          <Typography.Title className="mt-0 mb-8" level={4}>
-            Path
-          </Typography.Title>
-          {markdownContent && (
+    <>
+      <Meta
+        title={pathData.name}
+        description={pathData.description}
+        url={pathData.service.url}
```
Comment (Tanya-atatakai) [inline pages/paths/[id].jsx:80]: this does't seem correct, probably should contain build domain + paths/{id}
image

File: components/Meta.jsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  const pageTitle = title ? `${title} | ${SITE_TITLE}` : SITE_TITLE;
+  return (
+    <Head>
+      <title>{pageTitle}</title>
+
+      <meta name="title" content={pageTitle} />
+      <meta name="description" content={description || SITE_DESCRIPTION} />
+
+      <meta property="og:type" content="website" />
+      <meta property="og:url" content={url || SITE_URL} />
```
Comment (Tanya-atatakai) [inline components/Meta.jsx:19]: maybe would be easier to do it like you do with title, so we don't repeat the origin url every time?: 
```
content={`${SITE_URL}/${path}}
```
(renamed url to path so it's clearer it's not the base url)

**Review by Tanya-atatakai — CHANGES_REQUESTED** [review]

**Review by truemiller — APPROVED** [review]

**Review by oaksprout — APPROVED** [review]

**Pull request merged**